SchoolProject/src/main/java/commands/teacherCommands/ListStudentsByGradeCommandImp.java

```diff
@@ -0,0 +1,22 @@
+package commands.teacherCommands;
+
+import commands.ITeacherOperations;
+import commands.TeacherReciver;
+import helpers.Grade;
+
+public class ListStudentsByGradeCommandImp implements ITeacherOperations {
```

Comment:
I like this idea... but It is probably that your menu options increase, so, you will create another command to do that...